src/user/user.action.js

```diff
@@ -85,13 +85,36 @@ export const checkFollowStatus = url => {
   };
 };
 
+export const getFollowers = user => {
+  return (dispatch, getState) => {
+    const accessToken = getState().auth.accessToken;
+
+    dispatch({ type: GET_FOLLOWERS.PENDING });
+
+    fetchUrl(`${USER_ENDPOINT(user.login)}/followers?per_page=100`, accessToken)
+      .then(data => {
+        dispatch({
+          type: GET_FOLLOWERS.SUCCESS,
+          payload: data,
+        });
+      })
+      .catch(error => {
+        dispatch({
+          type: GET_FOLLOWERS.ERROR,
+          payload: error,
+        });
+      });
+  };
+};
+
 export const getUserInfo = user => {
   return dispatch => {
     return dispatch(getUser(user)).then(() => {
       dispatch(getOrgs(user));
       dispatch(
         checkFollowStatus(`https://api.github.com/user/following/${user}`)
       );
+      dispatch(getFollowers(user));
```

Comment:
exactly! I'll remove that action on `follower-list.screen`. Thanks for pointing out 👍